opendbc/car/__init__.py

```diff
@@ -7,14 +8,24 @@
 from opendbc.car import structs, uds
 from opendbc.car.can_definitions import CanData
 from opendbc.car.docs_definitions import CarDocs, ExtraCarDocs
+from opendbc.car.vehicle_model import VehicleModel
 
 DT_CTRL = 0.01  # car state and control loop timestep (s)
 
 # kg of standard extra cargo to count for drive, gas, etc...
 STD_CARGO_KG = 136.
 
+# ISO 11270
+ISO_LATERAL_ACCEL = 3.0  # m/s^2
+ISO_LATERAL_JERK = 5.0  # m/s^3
```

Comment:
these are already defined in opendbc.car.interfaces

Reply:
I could swear I had removed them from there..

Reply:
Gone. it is no longer on interfaces, I wanted to remove it form there, otherwise it'd be a circular dependency tho